docs/basic_training/channels.pt.md

```diff
@@ -0,0 +1,696 @@
+  log.info "${row[0]} -- ${row[2]}"
+}
+```
+
+!!! exercise
+
+    Tente inserir leituras fastq no fluxo de trabalho do RNA-Seq anterior usando `.splitCSV`.
+
+    ??? solution
+
+        Adicione um arquivo de texto csv contendo o seguinte, como uma entrada de exemplo com o nome "fastq.csv":
+
+        ```csv
+        gut,/workspace/gitpod/nf-training/data/ggal/gut_1.fq,/workspace/gitpod/nf-training/data/ggal/gut_2.fq
+        ```
+
+        Em seguida, substitua o canal de entrada para as leituras em `script7.nf`, alterando as seguintes linhas:
+
+        ```groovy linenums="1"
+        Channel
+          .fromFilePairs( params.reads, checkIfExists: true )
+          .set { read_pairs_ch }
+        ```
+
+        Para uma entrada de fábrica de canal splitCsv:
+
+        ```groovy linenums="1" hl_lines="2 3 4"
+        Channel
+          .fromPath("fastq.csv")
+          .splitCsv()
+          .view () { row -> "${row[0]},${row[1]},${row[2]}" }
+          .set { read_pairs_ch }
+        ```
+
+        Por fim, altere a cardinalidade dos processos que usam os dados de entrada. Por exemplo, para o processo de quantificação, mude de:
+
+        ```groovy linenums="1"
+        process QUANTIFICATION {
+          tag "$sample_id"
+
+          input:
+          path salmon_index
+          tuple val(sample_id), path(reads)
+
+          output:
+          path sample_id, emit: quant_ch
+
+          script:
+          """
+          salmon quant --threads $task.cpus --libType=U -i $salmon_index -1 ${reads[0]} -2 ${reads[1]} -o $sample_id
+          """
+        }
+        ```
+
+        Para:
+
+        ```groovy linenums="1" hl_lines="6 13"
+        process QUANTIFICATION {
+          tag "$sample_id"
+
+          input:
+          path salmon_index
+          tuple val(sample_id), path(reads1), path(reads2)
+
+          output:
+          path sample_id, emit: quant_ch
+
+          script:
+          """
+          salmon quant --threads $task.cpus --libType=U -i $salmon_index -1 ${reads1} -2 ${reads2} -o $sample_id
+          """
+        }
+        ```
+
+        Repita o procedimento acima para a etapa fastqc.
+
+        ```groovy linenums="1"
+        process FASTQC {
+          tag "FASTQC on $sample_id"
+
+          input:
+          tuple val(sample_id), path(reads1), path(reads2)
+
+          output:
+          path "fastqc_${sample_id}_logs"
+
+          script:
+          """
+          mkdir fastqc_${sample_id}_logs
+          fastqc -o fastqc_${sample_id}_logs -f fastq -q ${reads1} ${reads2}
+          """
+        }
+        ```
+
+        Agora o fluxo de trabalho deve ser executado a partir de um arquivo CSV.
+
+### Valores separados por tabulação (.tsv)
+
+A análise de arquivos tsv funciona de maneira semelhante, basta adicionar a opção `sep:'\t'` no contexto `splitCsv`:
+
+```groovy linenums="1"
+Channel
+  .fromPath("data/meta/regions.tsv", checkIfExists:true)
+  // use `sep` option to parse TAB separated files
```

Comment:
Talvez traduzir os comentários? Não sei como tá o posicionamento a respeito de traduzir os elementos de código.

Reply:
Acho que vale uma conversa sobre! 
Algumas das seções que já foram traduzidas não incluíram traduções dos comentários e outros elementos de código. Acho que seria bacana traduzir, mas não sei se haveria algum impedimento, seja quando esse material for atualizado no futuro ou nas traduções que estão sendo feitas nas outras línguas.

Reply:
É algo que precisamos documentar, mas eu tenho utilizado a seguinte lógica: Traduz tudo que for possível, com uma única exceção: Código que está salvo em scripts no repositório (como os da seção de RNAseq). Por quê? Porque traduzir estes também implicaria em criar vários arquivos novos, dentre outras coisas. Mas fora essa exceção, acho que o correto é traduzir tudo mesmo (nome de variáveis, comentários, etc). Isso aí passou batido também! Vou corrigir agora.